app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
+
+# A method that updates the comments counter for a post.
+def update_comments_counter
+  Post.select.update(:comments_counter).count
+end
```

Comment:
- [ ] `update_comments_counter` method should be inside `Comment` class
- [ ] I suggest that you use [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `#update_comments_counter` method every time a new like has been saved to database, then make the `update_comments_counter` method private

Reply:
Done.